pkg/core/statesync/module.go

```diff
@@ -466,7 +479,8 @@ func (s *Module) Traverse(root util.Uint256, process func(node mpt.Node, nodeByt
 	s.lock.RLock()
 	defer s.lock.RUnlock()
 
-	b := mpt.NewBillet(root, s.bc.GetConfig().KeepOnlyLatestState, storage.NewMemCachedStore(s.dao.Store))
+	b := mpt.NewBillet(root, s.bc.GetConfig().KeepOnlyLatestState,
+		storage.STTempStorage, storage.NewMemCachedStore(s.dao.Store))
```

Comment:
I'd rather put some garbage prefix here because we shouldn't use it during Traverse anyway. Or we may add a comment.

Reply:
What do you mean by "some garbage prefix"? The idea of PR is to use one prefix as temprorary (garbage?) and the second as the actual state (possibly old, would be replaced after sync).